docs/source/conventions/python_conventions.rst

```diff
@@ -7,14 +7,39 @@ Python.
 
 .. All pull requests are expected to adhere to these conventions. 
 
+Code Formatting
+===============
+
+When possible, ``yapf`` (`PyPI link <https://pypi.org/project/yapf/>`__) will 
+be used to enforce code conventions. In general, Python code should be 
+`PEP 8 compliant <https://www.python.org/dev/peps/pep-0008/>`__.
+
+VSCode Integration
+------------------
+
+VSCode can use ``yapf`` to automatically format code through the `official 
+Microsoft Python extension <https://marketplace.visualstudio.com/items?
+itemName=ms-python.python>`__. Install ``yapf`` using ``pip install yapf``
+and set ``python.formatting.provider`` to "yapf" to use ``yapf`` when 
+auto-formatting your code.
+
+.. note:: 
+   
+   More information on VSCode Python formatting can be found `here 
+   <https://code.visualstudio.com/docs/python/editing#_formatting>`__.
```

Comment:
It's up to you, but I think it makes more sense to move this and the other VSCode Integration tip to [here](https://nwchemex-project.github.io/DeveloperTools/development/ides/vscode/developing.html) so that we keep the VSCode tips together. If you wanted to, you could link to those VSCode integrations from this page.

Reply:
Makes sense! I will move this over.